contracts/wrapper/MetaERC20Wrapper.sol

```diff
@@ -52,7 +47,7 @@ contract MetaERC20Wrapper is ERC1155Meta, ERC1155MintBurn {
    * @dev Deposit ETH in this contract to receive wrapped ETH
    * No parameters provided
    */
-  function() external payable {
+  fallback () external payable {
```

Comment:
Maybe it would be better to use `receive()` instead of `fallback() payable`, I think that the only difference is that `receive()` requires msg.data to be empty.